content/10-std.md

```diff
@@ -36,6 +36,15 @@ trace(a == b); // true
 
 The only difference between the two forms is that single-quoted literals allow [string interpolation](lf-string-interpolation).
 
+String literals may occupy multiple lines; In which case, each line of the string will be punctuated with a '\n' newline character (See the escape sequences chart below).
+
+```haxe
+var str = "Line one
+	Line two
+	Line three"; //Lines two and three have \t in them
```

Comment:
The point of this addition is to demonstrate new lines. I think it's better to show this without the tabs, it might be confusing. You could have one example where the second line onwards is not indented, then show that if you indent the string as you would the code in a function, e.g.

```
class X {
  function foo() {
    var str = "a
    b
    c";
  }
}
```

Then the indents become part of the string. Does that make sense?

Reply:
That makes sense; I've implemented that change in a new commit. Should I merge it with the previous one?

Reply:
PRs here are squashed, so it does not matter whether it is a separate commit.